sass/components/_gallery.scss

```diff
@@ -157,3 +157,230 @@ Style guide gallery
     height: auto;
   }
 }
+
+/*
+Gallery Overlay
+
+Gallery overlay component.
+
+Markup: html/gallery-overlay.html
+
+Style guide: gallery.overlay
+*/
+
+// Media outside container
+.gallery-overlay {
+  max-width: 1400px;
+  display: -webkit-flex;
+  display: flex;
+  flex-direction: column;
```

Comment:
The flex-direction property will not work for IE9, only works for IE11
https://www.w3schools.com/cssref/css3_pr_flex-direction.asp